static/app/types/hooks.tsx

```diff
@@ -65,6 +72,14 @@ type CodeOwnersHeaderProps = {
   addCodeOwner: () => void;
   handleRequest: () => void;
 };
+type FirstPartyIntegrationAlertProps = {
+  integrations: Integration[];
+  wrapWithContainer?: boolean;
+  hideCTA?: boolean;
+};
+type FirstPartyIntegrationAdditionalCTAProps = {
+  integrations: Integration[];
+};
```

Comment:
Can we space out some of these props. This stuff is all smushed together

Reply:
lol you and your new lines